ui/components/multichain/asset-picker-amount/asset-picker-modal/index.scss

```diff
@@ -0,0 +1,62 @@
+.asset-picker-modal {
+  $self: &;
+
+  position: absolute;
+  top: 5%;
+  left: 0%;
+  transform: translate(-50%, -50%);
+
+  &__search-list {
+    max-width: 100%;
+    overflow: auto;
+    max-height: 200px;
+  }
+
+  &__autodetect {
+    vertical-align: unset;
+  }
+}
+
+.multichain-asset-picker-list-item {
+  position: relative;
+  cursor: pointer;
+
+  &:not(.multichain-asset-picker-list-item--selected) {
+    &:hover,
+    &:focus-within {
+      background: var(--color-background-default-hover);
+    }
+  }
+
+  a:hover,
+  a:focus {
+    color: inherit;
+  }
+
+  &:hover,
+  &:focus,
+  &:focus-within {
+    .multichain-asset-picker-list-item__delete {
+      visibility: visible;
+    }
+  }
+
+  &__selected-indicator {
+    width: 4px;
+    height: calc(100% - 8px);
+    position: absolute;
+    top: 4px;
+    left: 4px;
+  }
+}
+
+.multichain-token-list {
```

Comment:
I think much of this CSS can be moved to `<Box>` props.

Reply:
done :)